File: aem-guides-wknd-spa-1.0.3-React/core/src/main/java/com/adobe/aem/guides/wknd/spa/react/core/models/MultiField.java
package com.adobe.aem.guides.wknd.spa.react.core.models;

import com.adobe.cq.export.json.ComponentExporter;

public interface MultiField extends ComponentExporter {
    public String getTextTitle();
    public String getHeading();
    public String getText();
    public String getImage();
}

File: aem-guides-wknd-spa-1.0.3-React/ui.frontend/src/components/MultiField/MultiField.js
import React from 'react';
import { MapTo } from '@adobe/aem-react-editable-components';

const MultiFieldConfig = {
    empty :'MultiField',
    isEmpty: function(props){
        return !props || !props.title
    }
};

const MultiField = (props) => {
  return (
    <div>MultiField</div>
  )
}

MapTo('wknd-spa-react/components/MultiField')(MultiField, MultiFieldConfig);
